frontend/src/components/Profile.jsx

```diff
@@ -0,0 +1,65 @@
+import React, {useEffect, useState} from "react";
+import Grid from "@mui/material/Grid";
+import Divider from "@mui/material/Divider";
+import {ethers} from "ethers";
+import TabPanel from "./common/TabPanel";
+import ProfileBanner from "./profile/ProfileBanner";
+import AccountInfo from "./profile/AccountInfo";
+import TabOptions from "./profile/TabOptions";
+import NFTList from "./profile/NFTList";
+import NoItems from "./profile/NoItems";
+
+const Profile = (props) => {
+
+  const {account} = props
+  const [balance,setBalance] = useState(0)
+  const [tabValue, setTabValue] = useState(1)
+
+
+
+  const handleTabChange = (event, newValue) => {
```

Comment:
we should think about using useCallback for these ones; non-blocking right now tho

Reply:
pls teach me :3

Reply:
```suggestion
  const handleTabChange = useCallback(function ..., [dependencies])
```
basically since functions don't really change, we can useCallback to sort of remember the function. That way when the component re-renders based on some other field changing, the function wrapped in useCallback won't be re-rendered